developer/src/tike/xml/project/welcome.xsl

```diff
@@ -80,7 +80,7 @@
         <xsl:with-param name="caption">Open</xsl:with-param>
         <xsl:with-param name="command">keyman:openfile?id=<xsl:value-of select="ID" /></xsl:with-param>
       </xsl:call-template>
-      <xsl:if test="FileType='.kmn' or FileType='.kps' or FileType='.kpp'">
+      <xsl:if test="FileType='.kmn' or FileType='.kps' or FileType='.xml-ldml-keyboard'">
```

Comment:
Is `.kpp` deprecated? I don't remember ever seeing it.

Reply:
It's the old 'Keyman Product' format, for building standalone Windows products based on Keyman. A leftover from the bad old days...